### DISCUSSION TITLE
Sleeping bag Upgrade

### DISCUSSION THREAD

**Discussion by PigeonNian** [P3pp3rF1y/SophisticatedBackpacks#230] [Ideas]
Add a upgrade that can build a bed or sleeping bag, just like Traveler's Backpack.
Maybe is difficult to  realize but it will be a convenient function for travel.

**Comment by LeoColman** [comment]
I'd use that!

**Comment by Noobly-Walker** [comment]
Maybe also an advanced version that contains not just a sleeping bag, but a whole tent. Place the backpack down and toggle it, and you have a small room, safe from the mobs outside. The backpack already includes ludicrous storage for not only items, but also fluid and energy, and has various crafting stations built-in. Might as well make it a backpack you can live in.

The bed and tent both should be the same color as the backpack, or white if the backpack wasn't dyed.
